content/en/tracing/api_catalog/_index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Datadog API Catalog
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+{{< beta-callout url="https://forms.gle/TgezmEhUPXzebvv27" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
+  Datadog API Catalog is in beta! Use this form request access. 
+{{< /beta-callout >}} 
+
+{{< img src="tracing/api_catalog/api-catalog-overview.png" alt="API Catalog showing a list of API endpoints along with their Error Rate, P99 Latency, Requests Count, and Owning Team, with searching, filtering, and scoping features." style="width:100%;" >}}
+
+## Overview
+
+The API Catalog provides a single view and entry point into exploring the performance, reliability, and ownership of all your API Endpoints in one place. It's a central place where your whole company can find constantly up-to-date information about the characteristics of the APIs used by internal services (private APIs) and external users (publicly exposed APIs). 
+
+Monitor your mission-critical API-driven business features by, standardize APIs performance expectations and constantly validate these expectations, alerting when performance deviates from them.
+
+This enables your efforts to:
+- Provide high availability and uptime of critical APIs and business features that relies on them to function.
+- Have fewer API regressions and instability issues.
+- Triage incidents quicker.
+
+API Catalog brings together data from across Datadog to provide opinionated workflows so you can explore and monitor your APIs from different sources in one unified view. API Catalog provides:
+
+- **Automated discoverability** - One inventory for all public, private, and partner APIs, where _endpoints_ are the organizing principle.
+- **Correlation of all API metadata** from different Datadog sources.
+- **API endpoint metrics** such as Last Seen, Requests, Latency, and Errors, to identify performance issues and track API health.
+- **Alerting** on Endpoints that deviate from defined performance expectations and thresholds.
+- Resolving incidents faster with API **ownership information** (team, on-call, communication channels) directly associated with each endpoint, to know who to reach when something goes wrong
+
+<div class="alert alert-info">Not sure what an API or an endpoint is? See <a href="#key-terminology">Key terminology</a> for some background on concepts that are important to understanding what API Catalog does.</div>
```

Comment:
I think someone wouldn't be here if they didn't know what an API or endpoint is. Maybe we can rephrase this to be a pointer to the glossary and not so basic.